Sprint1.md

```diff
@@ -0,0 +1,22 @@
+The goal for the Sprint 1 was to develop the essential Front-End Pages, API's and set-up the DB along with tables.
+
+Following REST API’s have been created :
+
+- Retrieve all Users : GET /api/users
+- Retrieve User based on User Id : GET /api/users/id/:id
+- Retrieve User based on Email Id : GET /api/users/email/:email
+- Add a New User : POST /api/users/newuser
+- Update User Details : POST /api/users/newuser
```

Comment:
It should be:

Update User Details: PUT /api/users/:id

I have corrected the error in API reference as well